### ISSUE TITLE
run_pretraining的hparams的参数能共享下吗？

### ISSUE THREAD

**Issue by luoyexuge** [ymcui/Chinese-ELECTRA#16]
在跑run_pretraining的时候碰到
logits = tf.matmul(hidden, model.get_embedding_table(),
                           transpose_b=True)
logits = tf.nn.bias_add(logits, output_bias)

这行报错，原因是hidden是的一个rank=3的tensor， model.get_embedding_table()是一个等于2的tensor，不知道有没有碰到这样的错误？

**Comment by ymcui** [comment]
没有碰到过。
run_pretraining.py的hparams参数如下（以small为例）：

```
"pretrain_tfrecords":"your_data.tfrecord*", 
"vocab_file":"vocab.txt", 
"model_dir":"./models/small", 
"vocab_size": 21128,
"max_seq_length": 512,
"model_size": "small",
"num_train_steps": 1000000,
"save_checkpoints_steps": 10000,
"train_batch_size": 1024,
```

其余默认参数存放在`configure_pretraining.py`文件。

**Comment by ymcui** [comment]
如有必要，可随时reopen。

**Comment by cuixuage** [comment]
> 在跑run_pretraining的时候碰到
> logits = tf.matmul(hidden, model.get_embedding_table(),
> transpose_b=True)
> logits = tf.nn.bias_add(logits, output_bias)
> 
> 这行报错，原因是hidden是的一个rank=3的tensor， model.get_embedding_table()是一个等于2的tensor，不知道有没有碰到这样的错误？

你好，请问后来这个问题你解决了吗？
